src/types/field/shortTextField.ts

```diff
@@ -11,4 +11,7 @@ export interface IShortTextField extends IField {
   ValidationOptions: ShortTextValidationOptions
 }
 
-export interface IShortTextFieldSchema extends IShortTextField, IFieldSchema {}
+export interface IShortTextFieldSchema extends IShortTextField, IFieldSchema {
+  // Prefill flag
+  allowPrefill?: boolean
```

Comment:
should this flag not be optional?
```suggestion
  allowPrefill: boolean
```

Reply:
fair point, edited accordingly!

Reply:
no it should be optional, since this controls the params needed to instantiate the model. If required then all instantiations of the field needs to have the `allowPrefill` value.